apps/src/applab/dataBrowser/AddColumnButton.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * @overview Component for adding a new column to the specified table.
+ */
+
+import Radium from 'radium';
+import React from 'react';
+
+import * as dataStyles from './dataStyles';
+import { connect } from 'react-redux';
+
+const containerStyle = {
+  float: 'right',
+  paddingTop: 10,
+  paddingLeft: 10,
+  paddingBottom: 10,
+  paddingRight: 0
+};
+
+const AddColumnButton = React.createClass({
+  propTypes: {
+    addColumn: React.PropTypes.func.isRequired
+  },
+
+  handleAdd() {
+    this.props.addColumn();
+  },
+
+  render() {
+    return (
+      <div style={containerStyle}>
+        <button className="btn" onClick={this.handleAdd} style={dataStyles.button}>
+          Add column
+        </button>
+      </div>
+    );
+  }
+});
+export default Radium(AddColumnButton);
```

Comment:
Hi Brad, I agree that this react class is a little light right now. However, the next feature on my plate is to add Import, Export and Delete buttons to this same row, and I had secretly been planning to rename this class to contain the entire top row of buttons (import, export, add column, delete table):
<img width="455" alt="screen shot 2016-07-25 at 4 04 57 pm" src="https://cloud.githubusercontent.com/assets/8001765/17120503/9516733e-5281-11e6-827a-bf38501f2ee6.png">

I've renamed the file and removed some cruft.